main.py

```diff
@@ -1,10 +1,71 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
```

Comment:
This is a code quality issue. You are opening a file 'players.json' and reading it inside the main function. This is not a good practice because it makes your function less flexible and harder to test. It would be better to move the file reading to a separate function and pass the loaded data to the main function.